js/packages/eyes-testcafe/package.json

```
@@ -30,6 +30,27 @@
   ],
   "main": "./dist/index-legacy.js",
   "types": "./types/index-legacy.d.ts",
+  "typesVersions": {
+    "*":  {
+      "next": [
+        "./types/index.d.ts"
+      ]
+    }
+  },
+  "exports" : {
+    ".": {
+      "types": "./types/index-legacy.d.ts",
+      "default": "./dist/index-legacy.js"
+    },
+    "./dist": {
+      "types": "./types/index.d.ts",
+      "default": "./dist/index.js"
+    },
```

Also, I would allow importing any file from `dist`.
```suggestion
    "./dist/*": {
        "types": "./types/*.d.ts",
        "default": "./dist/*.js"
    },
```